mentat/app.py

```diff
@@ -35,10 +36,25 @@ def run_cli():
         default=[],
         help="List of file paths, directory paths, or glob patterns to exclude",
     )
+    parser.add_argument(
+        "--revert",
+        action="store_true",
+        help="Restore from the mentat_backups directory",
+    )
     args = parser.parse_args()
-    paths = args.paths
-    exclude_paths = args.exclude
-    run(expand_paths(paths), expand_paths(exclude_paths))
+
+    backup_dir = ".mentat_backups"
+
+    if args.revert:
+        revert_files(backup_dir)
+    else:
+        paths = args.paths
+        exclude_paths = args.exclude
+        run(
+            expand_paths(paths),
+            expand_paths(exclude_paths),
+            backup_dir,
```

Comment:
Let's get rid of the backup_dir variable up here and in the run function; it feels a bit weird to have 3 default values for it (one in the run function, one up here, and one in the constructor for backupmanager).